jib-core/src/main/java/com/google/cloud/tools/jib/http/FailoverHttpClient.java

```diff
@@ -272,13 +272,13 @@ public Response call(String httpMethod, URL url, Request request) throws IOExcep
 
       try {
         logInsecureHttpsFailover(url);
-        Response response = call(httpMethod, url, request, getHttpTransport(false));
+        Response response = call(httpMethod, url, request, getHttpTransport(false), false);
```

Comment:
Not a terminal call, we may have to failover to plain HTTP once more, so disable retrying.

Reply:
A comment mentioning this could be helpful for future readers. WDYT?

Reply:
Good idea. Since the argument is to be removed eventually, and also for brevity, added a link to this PR in one of the comments.